src/types.rs

```diff
@@ -231,6 +231,8 @@ pub struct ZipFileData {
     pub header_start: u64,
     /// Specifies where the compressed data of the file starts
     pub data_start: u64,
+    /// Internal file attributes
+    pub internal_attributes: u16,
```

Comment:
This is out of scope for this PR but I wonder if keeping this internally as a bitflags enum would be better....